bravado_asyncio/http_client.py

```diff
@@ -145,7 +154,7 @@ def request(self, request_params, operation=None, response_callbacks=None,
             also_return_response,
         )
 
-    def prepare_params(self, params):
+    def prepare_params(self, params: Optional[Dict[str, Any]]) -> MultiDict:
```

Comment:
I do expect that ``params`` need to be a dictionary ... how this could be ``None``?

Reply:
It's defensive programming that I've copied from the fido client. If we do `request_params.get('params') above then the method definition needs to say `Optional`, as the return type of dict.get() is an Optional. I checked the bravado source and it is indeed always a dict. Do we want to assume that it is?

Reply:
Thanks for the explanation ... I'm fine either case